api/mapping/services.py

```diff
@@ -146,16 +146,6 @@ def process_scan_report(scan_report_id):
                     else:
                         scanreport.is_date_event = False
 
-                    if scanreport.ignore_column == "Ignore":
```

Comment:
We still need this logic to build a more concise data dictionary for NLP. Can you reinstate this code, please? Thanks!